### ISSUE TITLE
[bug]: post-logo focus styles are not accessible

### ISSUE THREAD

**Issue by gfellerph** [swisspost/design-system#4417]
**Describe the bug**
Post logo has the default focus styles and they're not accessible in the header.

**To Reproduce**

![Image](https://github.com/user-attachments/assets/0a2d4fec-dabf-431e-b097-355f2b5ab4e3)

1. Focus the logo

**Expected behavior**
Logo should have the same focus outline like the rest of the application and the focus outline should be entirely visible.

**Desktop:**

- OS: Windows
- Browser: chrome
